src/main/webapp/WEB-INF/views/admin/edit-room.jsp

```diff
@@ -71,17 +71,15 @@
                     <c:url var="lockUrl" value="/adm-edit-room?id=${room.id}"/>
                     <form:form id="${managerFormId}" action="${lockUrl}" method="POST" >
                         <input id="room" name="manager" type="hidden" value="${room.id}" />
-                        <button type="submit" value="lock"
-                                onClick="return confirm('Are you sure you want to make the room active?')"
+                        <button id = "activateButton" type="button" value="lock"
                                 class="button button-size-default delete"></button>
                     </form:form>
                 </c:if>
                 <c:if test="${room.active eq true}">
                     <c:url var="lockUrl" value="/adm-edit-room?id=${room.id}"/>
                     <form:form id="${managerFormId}" action="${lockUrl}" method="POST" >
                        <input id="room" name="room" type="hidden" value="${room.id}" />
-                       <button type="submit" value="lock"
-                               onClick="return confirm('Are you sure you want to make the room inactive?')"
+                       <button id = "deactivateButton" type="button"
```

Comment:
It is working correctly. I use the same handler (method click) on every item with this id. And every time when user press on this element i catch it by using $(this).